src/skills-builder/utils/extractProductKeys.js

```diff
@@ -30,10 +30,11 @@ export const extractProductKeys = (recommendations, expandedList = []) => (
       return [
         type,
         // create an array of objects for each product line
-        // each object contains a title and product key, rather than the entire response from Algolia (rec)
-        recommendationsRefinedList?.map(rec => ({
+        // each object contains a title, product label, and position - rather than the entire response from Algolia
+        recommendationsRefinedList?.map((rec, index) => ({
           title: rec.title,
-          courserun_key: rec.active_run_key,
+          label: rec.active_run_key || `${rec.title} [${rec.uuid}]`,
```

Comment:
This little bit of naming logic is duplicated and also important. Should we add a comment that if the `label` is changed that we should also update `productLabel` in `RecommendationCard.jsx` (and vice-versa)?

Not sure if it would ever change... but seems like it would cause a weird mismatch and may be hard to track down?

Reply:
Good call out! I'll add a comment in both places. The schema is bound to be modified again so it'll be nice to have the reminder